packages/components/pie-radio/src/index.ts

```diff
@@ -18,19 +19,19 @@ const componentSelector = 'pie-radio';
  * @tagname pie-radio
  */
 export class PieRadio extends FormControlMixin(RtlMixin(LitElement)) implements RadioProps {
-    @property({ type: Boolean })
+    @property({ type: Boolean, reflect: true })
     public checked = defaultProps.checked;
 
-    @property({ type: Boolean })
+    @property({ type: Boolean, reflect: true })
     public defaultChecked = defaultProps.defaultChecked;
 
-    @property({ type: Boolean })
+    @property({ type: Boolean, reflect: true })
     public disabled = defaultProps.disabled;
 
     @property({ type: String })
     public name: RadioProps['name'];
 
-    @property({ type: Boolean })
+    @property({ type: Boolean, reflect: true })
```

Comment:
could you please share our current assumptions about when a property should reflect or not? From a quick search, I noticed that some form components have reflect: true for the required prop, while others do not 👀